react/AsidedLayout/AsidedLayout.js

```diff
@@ -0,0 +1,30 @@
+import styles from './AsidedLayout.less';
+import React from 'react';
+import classnames from 'classnames';
+
+const defaultRenderAside = () => null;
+
+export default function AsidedLayout({ className, children, renderAside = defaultRenderAside, size, reverse }) {
+  return (
+    <div
+      className={classnames({
+        [className]: className,
```

Comment:
Just to be clear, the order of classes in a `className` prop has no impact on specificity. I decided to put `className` first to make it obvious that styles can be overridden, but happy to settle on the opposite convention. What do you think?